example/next.config.js

```diff
@@ -4,18 +4,21 @@ module.exports = {
   webpack(config) {
     config.resolve.alias.react = path.resolve(
       __dirname,
-      './node_modules/react',
+      '../node_modules/react/',
     );
     config.resolve.alias['react-dom'] = path.resolve(
       __dirname,
-      './node_modules/react-dom',
+      '../node_modules/react-dom/',
     );
     config.resolve.alias['react-is'] = path.resolve(
       __dirname,
-      './node_modules/react-is',
+      '../node_modules/react-is/',
     );
-    config.resolve.alias.urql = path.resolve(__dirname, './node_modules/urql/');
-    config.output.pathinfo = true;
+    config.resolve.alias.urql = path.resolve(
+      __dirname,
+      '../node_modules/urql/',
+    );
+    config.resolve.alias['next-urql'] = path.resolve(__dirname, '../');
```

Comment:
I got rid of our tarball approach and instead just use `resolve.alias`. Interestingly, we still can't have local copies of `react` and `react-dom` in the examples because the repo's `react-ssr-prepass` will resolve the root `node_modules/react` dev dependency, leading to the two versions of React non-sense :facepalm: